app/org/sagebionetworks/bridge/dynamodb/DynamoScheduledActivity.java

```diff
@@ -88,22 +95,26 @@ public void setTimeZone(DateTimeZone zone) {
 
     @Override
     @DynamoDBIgnore
+    @JsonSerialize(using = DateTimeSerializer.class)
     public DateTime getScheduledOn() {
         return getInstant(getLocalScheduledOn());
     }
 
     @Override
+    @JsonDeserialize(using = DateTimeDeserializer.class)
     public void setScheduledOn(DateTime scheduledOn) {
         this.localScheduledOn = (scheduledOn == null) ? null : scheduledOn.toLocalDateTime();
     }
 
     @Override
     @DynamoDBIgnore
+    @JsonSerialize(using = DateTimeSerializer.class)
     public DateTime getExpiresOn() {
         return getInstant(getLocalExpiresOn());
     }
 
     @Override
+    @JsonDeserialize(using = DateTimeDeserializer.class)
```

Comment:
Okay, the deserialize values are not needed, these are never deserialized. They are used when calculating dates during scheduling, and the local value is stored separately from the time zone (which is set separately). Removed the annotations.